Source/cursor.cpp

```diff
@@ -237,7 +240,7 @@ void CheckRportal()
 				if (!setlevel)
 					strcpy(tempstr, _("The Unholy Altar"));
 				else
-					strcpy(tempstr, _("level 15"));
+					strcpy(tempstr, LvlNames[15]);

```

Comment:
Translation support 🙂 

```suggestion
					strcpy(tempstr, _(LvlNames[15]));
```